athena/map/Vector.tsx

```diff
@@ -104,6 +106,22 @@ export default abstract class Vector {
     return szudzik(this.x + 1, this.y + 1);
   }
 
+  mirror(
+    mapSize: SizeVector,
```

Comment:
There is some repetition in this function, so maybe just destructure like `{ height, width }: SizeVector`?

Reply:
Actually, since this function is only used sparely, I would probably prefer making it a standalone function in `athena/lib` or `hera/editor/lib` as like `mirrorVector(vector: Vector, mirrorType: MirrorType)`.